ck-editor/assets/index.html

```diff
@@ -5,13 +5,16 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="stylesheet" type="text/css" href="styles.css">
+		<script src="purify.min.js"></script>
```

Comment:
Can you describe the threat model here and what dompurify is guarding against?

Reply:
The threat model would be a source document with a crafted payload.
It is the same approach we take when converting .md to .html in the markdown viewer. It is just another layer to make sure that any code that might otherwise get executed by the html renderer is removed.

Reply:
So the threat actors here are the user (runner of the app), the app author, and the author of the document being viewed?

Reply:
in this case, it is the author of the document